lineman/app/app.coffee

```diff
@@ -17,10 +17,14 @@ angular.module('loomioApp', ['ngRoute',
 angular.module('loomioApp').config ($translateProvider) ->
   $translateProvider.
     useUrlLoader('/api/v1/translations/en').
-    preferredLanguage('en')
+    preferredLanguage('pt-BR')
```

Comment:
If it's bad form to access current_user in here, we might have to fetch and set locale post - initilization